src/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {App} from './components/app/app.jsx';
+
+const init = () => {
+  const settings = {
+    rentProposals: 1337,
+  };
+
+  ReactDOM.render(
+      <App rentProposals={settings.rentProposals} />,
```

Comment:
когда у вас появится достаточно много пропсов (или вы будете получать данные с сервера), такая запись будет занимать много места.
Как вариант:
```suggestion
      <App {...settings} />,
```